src/pages/cart/cart-items.hbs

```diff
@@ -1,5 +1,83 @@
-<ul>
-  {{#each cartItems}}
-  <li>{{product_name}}</li>
-  {{/each}}
-</ul>
\ No newline at end of file
+<div class="cart-items" data-template="cart-items">
+  <h2 class="heading heading--section">Items in Cart</h2>
```

Comment:
As per BEM, should this be `.cart-items__heading`?

Reply:
True

Reply:
There are other places it was used like this so I just copied and pasted
like in 
`cart-moneyBox.hbs`
`product-sort.hbs`
`product-filter.hbs`

Do you still want me to changes?
if yes 
is this ok?
```
<h2 class="cart-items__heading heading--section">Items in Cart</h2>
```